src/main/java/tools/jackson/databind/introspect/AnnotatedMethod.java

```diff
@@ -73,27 +123,56 @@ public Class<?> getRawType() {
 
     @Override
     public final Object call() throws Exception {
-        // 31-Mar-2021, tatu: Note! This is faster than calling without arguments
-        //   because JDK in its wisdom would otherwise allocate `new Object[0]` to pass
-        return _method.invoke(null);
+        try {
+            return _nullaryInvoker.invokeExact();
+        } catch (final Throwable e) {
+            throw sneakyThrow(e);
```

Comment:
I guess MethodHandle.invokeXxx() report throwing `Throwable`, but is there anything beyond `Exception` and `Error` (unchecked)?

Reply:
No, it just is passing through whatever is thrown downstream - there should be no 'new' dynamically exception throwable here, all the access checks were done at MethodHandle time.